Debugging Fundamentals/Task1/Product.cs

```diff
@@ -7,6 +9,19 @@ public Product(string name, double price)
             Name = name;
             Price = price;
         }
+        public override bool Equals(object obj)
+        {
+            if (obj == null || obj.GetType() != this.GetType())
+                return false;
+            Product other = (Product)obj;
```

Comment:
Please provide meaningful names to all variables.

Reply:
resolved